interfaces/matlab_experimental/Kinetics/Kinetics.m

```diff
@@ -0,0 +1,536 @@
+            %    specified, ``species`` must be specified as well.
+            % :return:
+            %    Returns a sparse matrix of all reactant stoichiometric
+            %    coefficients. The matrix elements ``nu(k, i)`` is the
+            %    stoichiometric coefficient of species k as a reactant in
+            %    reaction i. If ``species`` and ``rxns`` are specified, the
+            %    matrix will contain only entries for the specified species
+            %    and reactions. For example, ``stoich_p(a, 3, [1, 3, 5,
+            %    7])`` returns a sparse matrix containing only the
+            %    coefficients for specis 3 in reactions 1, 3, 5, and 7.
+
+            nsp = kin.nTotalSpecies;
+            nr = kin.nReactions;
+            temp = sparse(nsp, nr);
+
+            if nargin == 1
+                krange = 1:nsp;
+                irange = 1:nr;
+            elseif nargin == 3
+                krange = species;
+                irange = rxns;
+            else error('stoichReactant requires 1 or 3 arguments.')
+            end
+
+            for k = krange
+
+                for i = irange
+                    t = ctFunc('kin_reactantStoichCoeff', ...
+                                kin.kinID, k - 1, i - 1);
+
+                    if t ~= 0.0
+                        temp(k, i) = t;
+                    end
+
+                end
+
+            end
+
+            n = temp;
+        end
+
+        function n = stoichProduct(kin, species, rxns)
+            % Product stoichiometric coefficients.
+            %
+            % :param species:
+            %    Species indices for which product stoichiometric
+            %    coefficients should be retrieved. Optional argument; if
+            %    specified, ``rxns`` must be specified as well.
+            % :param rxns:
+            %    Reaction indicies for which product stoichiometric
+            %    coefficients should be retrieved. Optional argument; if
+            %    specified, ``species`` must be specified as well.
+            % :return:
+            %    Returns a sparse matrix of all product stoichiometric
+            %    coefficients.
+
+            nsp = kin.nTotalSpecies;
+            nr = kin.nReactions;
+            temp = sparse(nsp, nr);
+
+            if nargin == 1
+                krange = 1:nsp;
+                irange = 1:nr;
+            elseif nargin == 3
+                krange = species;
+                irange = rxns;
+            else error('stoichProduct requires 1 or 3 arguments.')
+            end
+
+            for k = krange
+
+                for i = irange
+                    t = ctFunc('kin_productStoichCoeff', ...
+                                kin.kinID, k - 1, i - 1);
+
+                    if t ~= 0.0
+                        temp(k, i) = t;
+                    end
+
+                end
+
+            end
+
+            n = temp;
+        end
+
+        function n = stoichNet(kin, species, rxns)
```

Comment:
While there isn't a corresponding function in Python, this should be `netStoichCoeffs`.